go.mod

```diff
@@ -11,10 +11,12 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.15.2
 	github.com/ipfs/go-cid v0.0.7
+	github.com/lib/pq v1.8.0 // indirect
```

Comment:
revert

Reply:
it has been added automatically. I've noticed it already several times.

Reply:
removing.